<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEQRASMapperMainWindow</class>
 <widget class="QMainWindow" name="FEQRASMapperMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEQ RAS mapper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="nodeTablePushButton">
        <property name="text">
         <string>Node Table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nodeTableLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="rasDirPushButton">
        <property name="text">
         <string>RAS Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rasDirLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="feqSpecOutputGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>FEQ Special Output</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="addSpecOutputPushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editSpecOutputPushButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearSpecOutputPushButton">
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="specOutputListWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Map Export</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="rasmapFilePushButton">
         <property name="text">
          <string>RASMAP file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="rasmapFileLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="exportDBPushButton">
         <property name="text">
          <string>Export Database File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="exportDBLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cache Level</string>
         </property>
         <property name="buddy">
          <cstring>cacheLevelComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cacheLevelComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="planNameLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Plan Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="planNameComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Export Time Series</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of Days</string>
         </property>
         <property name="buddy">
          <cstring>numDaysLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numDaysLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Time Step</string>
         </property>
         <property name="buddy">
          <cstring>timeStepComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="timeStepComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="exportTilesPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Export Tiles for Web Viewing</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nodeTablePushButton</tabstop>
  <tabstop>nodeTableLineEdit</tabstop>
  <tabstop>addSpecOutputPushButton</tabstop>
  <tabstop>editSpecOutputPushButton</tabstop>
  <tabstop>clearSpecOutputPushButton</tabstop>
  <tabstop>specOutputListWidget</tabstop>
  <tabstop>numDaysLineEdit</tabstop>
  <tabstop>timeStepComboBox</tabstop>
  <tabstop>rasDirPushButton</tabstop>
  <tabstop>rasDirLineEdit</tabstop>
  <tabstop>rasmapFilePushButton</tabstop>
  <tabstop>rasmapFileLineEdit</tabstop>
  <tabstop>exportDBPushButton</tabstop>
  <tabstop>exportDBLineEdit</tabstop>
  <tabstop>cacheLevelComboBox</tabstop>
  <tabstop>exportTilesPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
